releaf-core/spec/lib/error_formatter_spec.rb

```diff
@@ -60,34 +67,38 @@ def base_validation
     end
 
     it "correctly adds errors for fields resource fields" do
-      expect( subject.errors["resource[title]"] ).to eq [blank_error]
+      expect( subject.errors["resource[title]"] ).to eq [blank_error('title')]
     end
 
     it "correclty adds errors for missing associated object (belongs_to)" do
-      expect( subject.errors["resource[author_id]"] ).to eq [blank_error]
+      expect( subject.errors["resource[author_id]"] ).to eq [blank_error('author')]
     end
 
     it "correctly adds error for missing associated object attributes (belongs_to)" do
       book.build_author
       book.valid?
-      expect( subject.errors["resource[author_attributes][name]"] ).to eq [blank_error]
+      expect( subject.errors["resource[author_attributes][name]"] ).to eq [blank_error('name', 'DummyResourceValidatorAuthor')]
     end
 
     it "correctly adds error for missing associated object attributes (has_many)" do
-      book.chapters.new
+      book.chapters.new(id: 12)
       book.chapters.new(:title => 'test')
       book.valid?
-      expect( subject.errors["resource[chapters_attributes][0][title]"] ).to eq [blank_error]
+      expect( subject.errors["resource[chapters_attributes][0][title]"] ).to eq [blank_error('title', 'Chapter', '12')]
       expect( subject.errors["resource[chapters_attributes][1][title]"] ).to be_nil
 
-      expect( subject.errors["resource[chapters_attributes][0][text]"] ).to eq [blank_error]
-      expect( subject.errors["resource[chapters_attributes][1][text]"] ).to eq [blank_error]
+      expect( subject.errors["resource[chapters_attributes][0][text]"] ).to eq [blank_error('text', 'Chapter', '12')]
+      expect( subject.errors["resource[chapters_attributes][1][text]"] ).to eq [blank_error('text', 'Chapter')]
```

Comment:
Line is too long. [111/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.